md-render/HHVM/UserDocumentation/IncludeExamplesFilter.rb

```diff
@@ -51,7 +51,21 @@ def call
             File.read(full_path),
             language: 'Hack',
           );
-          node.replace(code)
+          code_node = node.replace(code)
+          # If the example has HHVM output associated with it, then print it
+          # after the code example
+          if File.exists? (full_path + ".example.hhvm.out")
```

Comment:
> It would also be good if we could check that the .example.hhvm.out match the expectf/expectre

By running the test runner in process here for that particular example? Or some sort of regex check? All of our examples should pass the test runner. So if you produce the `.example.hhvm.out` via a run of the test runner, they would have to match because the test pass.